### ISSUE TITLE
Got error wile building a project. Cannot resolve dependency 'date-fns/isValid'

### ISSUE THREAD

**Issue by dvdvdmt** [mikeerickson/validatorjs#349]
Hello, I'm getting error while building a project: 
```/Users/dmitry/work/dev-to-dev/support-crm-client/node_modules/validatorjs/src/rules.js:1:22: Cannot resolve dependency 'date-fns/isValid'
> 1 | var isValid = require('date-fns/isValid');
    |                      ^
  2 | var parseISO = require('date-fns/parseISO');
  3 |
  4 | function leapYear(year) {
```
Check of npm package shows that there is not 'date-fns' in the dependencies.
```
npm view validatorjs@3.18.0 dependencies
{}
```

**Comment by namtrung804** [comment]
Same issue with me. try install `npm install date-fns` . that work with me

**Comment by lukas-pierce** [comment]
OK, but need move from `devDependencies` to `dependencies` in `package.json`

**Comment by vict-shevchenko** [comment]
It should be under `peerDependencies` in this case.

**Comment by smithaitufe** [comment]
`dependencies ` worked for me. `peerDependencies` failed

**Comment by iamrodrigo** [comment]
I rolled back to version 3.17.1 and this fixed the problem because I didn't want to install more dependencies. Hopefully the next release will fix this problem :)

**Comment by mikeerickson** [comment]
This issue has been resolved in the 3.18.1 patch posted today
